wazuh/wazuh-kibana-app
# Fix alerts summary reports

Hi team this PR solves some problems related to the Alert summary of the reports and also fix the last scan date of the CIS CAT module. Closes #3275 

### How to test
Check if the last scan date of the CIS CAT module is displayed as is set in the kibana settings. Also create some reports from vulnerabilities, Security Alerts and other modules to check if the number of alerts is correct.
